src/components/MainNav/MainNav.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC } from 'react';
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const MainNav: FC = () => (
+  <nav
+    data-cy="nav"
+    className="navbar is-fixed-top has-shadow"
+    role="navigation"
+    aria-label="main navigation"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
+          to="/"
+          className={({ isActive }) => classNames(
+            'navbar-item',
+            { 'has-background-grey-lighter': isActive },
+          )}
+        >
+          Home
+        </NavLink>
+        <NavLink
+          to="/people"
+          className={({ isActive }) => classNames(
+            'navbar-item',
+            { 'has-background-grey-lighter': isActive },
+          )}
+        >
+          People
+        </NavLink>
```

Comment:
maybe we can make a separate component for that, since they're pretty much identical:thinking: